webapp/channels/src/components/create_team/components/team_url/team_url.test.tsx

```diff
@@ -65,15 +50,28 @@ describe('/components/create_team/components/display_name', () => {
         const actions = {...defaultProps.actions, checkIfTeamExists};
         const props = {...defaultProps, actions};
```

Comment:
Out of the scope, but similar to my other comment about state, can be used for props. Here we can do this like...

```
const props = getDefaultProps();
props.actions.checkIfTeamExists = jest.fn().mockResolvedValueOnce({data: true}).mockResolvedValue({data: false});
```

Again, I feel it is less boilerplaty and more direct on what is being modified.